core/templates/services/utils.service.ts

```diff
@@ -33,11 +34,13 @@ export class UtilsService {
     return true;
   }
 
-  isString(input: any): boolean {
+  // The type of input is Object because it can be anything.
```

Comment:
Ditto.

Reply:
This can be inferred from the function itself. The function itself is made for checking the types of it's arguments. So, clearly the arguments could be anything.

Reply:
Yes, I got your point. Sorry, I think I confused here. I meant add a comment as follows:

```
The function here is to check whether the argument is a string. So, we cannot have any specific type defined for the argument and the argument is given a generic type of Object.
```

Reply:
Done!